src/tests/complex_tests/mod.rs

```diff
@@ -105,25 +105,25 @@ use crate::data_source::in_memory_data_source::InMemoryDataSource;
 use crate::witness::full_block_artifact::*;
 
 fn get_geometry_config() -> GeometryConfig {
-    // let geometry = crate::geometry_config::get_geometry_config();
-
-    GeometryConfig {
-        // cycles_per_vm_snapshot: 1,
-        cycles_per_vm_snapshot: 1024,
-        cycles_per_ram_permutation: 1024,
-        cycles_per_code_decommitter: 256,
-        cycles_per_storage_application: 4,
-        cycles_per_keccak256_circuit: 7,
-        cycles_per_sha256_circuit: 7,
-        cycles_per_ecrecover_circuit: 2,
-        // cycles_code_decommitter_sorter: 512,
-        cycles_code_decommitter_sorter: 3,
-        cycles_per_log_demuxer: 16,
-        cycles_per_storage_sorter: 16,
-        cycles_per_events_or_l1_messages_sorter: 4,
-
-        limit_for_l1_messages_pudata_hasher: 32,
-    }
+    crate::geometry_config::get_geometry_config()
+
+    // GeometryConfig {
```

Comment:
this private function is there for local testing, while global geometry config is for proving with much higher memory requirements